modules/release-notes/con-relnotes-notable-features.adoc

```diff
@@ -5,7 +5,9 @@ This section highlights new features in {product} {product-version}.
 
 == {product} Operator is now generally available (GA)
```

Comment:
Updating RN content is out of scope for this PR, but we need to create a Jira to remove this